timesketch/frontend-ng/src/views/Explore.vue

```diff
@@ -540,6 +551,32 @@ export default {
       chip.active = !chip.active
       this.search()
     },
+    copyChipValue(chip) {
+      let textToCopy = '';
+
+      // Different handling based on chip type
+      if (chip.type.startsWith('datetime')) {
+        // For datetime chips, just copy the value
+        textToCopy = chip.value;
+      } else {
+        // For other chips, copy both field and value
+        if (chip.operator === 'must_not') {
+          textToCopy = `NOT ${chip.field ? `${chip.field} : ` : ''}${chip.value}`;
+          } else {
+          textToCopy = `${chip.field ? `${chip.field} : ` : ''}${chip.value}`;
+        }
+      }
+
+      // Copy to clipboard
+      navigator.clipboard.writeText(textToCopy)
+        .then(() => {
+          this.copiedChip = chip;
+          setTimeout(() => this.copiedChip = null, 2000); // Reset after 2 seconds
+        })
+        .catch(err => {
+        // Handle errors (e.g., clipboard access denied)
+        });
```

Comment:
Please fix the linter errors here:
```
1. Arrow function should not return assignment
2. Expected error to be handled
```

See the GitHub workflow fore more details.